inexor/fpsgame/aiman.hpp

```diff
@@ -293,24 +270,18 @@ namespace aiman
         sendservmsg(msg);
     }
 
-	// notify bots that map has been changed
-	// force server bot manager to refresh bot balance
     void changemap()
     {
         dorefresh = true;
         loopv(clients) if(clients[i]->local || clients[i]->privilege) return;
         if(botbalance != (serverbotbalance != 0)) setbotbalance(NULL, serverbotbalance != 0);
     }
 
-	// a new human player connected
-	// change/refresh the way computer controlled players think
```

Comment:
is the second line correct? or is dorefresh a signal to the botmanager to e.g. remove bots?

maybe shrink that to one catchy line?